### DISCUSSION TITLE
SDRplay RSP1A not listed under the tuner section on an M1 MacBook.

### DISCUSSION THREAD

**Discussion by jrodh13** [DSheirer/sdrtrunk#1930] [Q&A] [answered]
New user here. Having trouble getting my SDRplay RSP1A working on an M1 Mac. I have followed the FAQ guidance and downloaded version 3.15 of the API and installed it. No tuners are recognized in the software although, SDRconnect recognizes it. Tried to reboot and even tried to run the commend to symbolically link the files to no avail. Any suggestions for a new user? Thanks in advance.

**Comment by DSheirer** [comment]
@jrodh13 I'll need some info to troubleshoot.

1. Please post the initial logging entries from the application log where it's going through the SDRPlay API and RSP tuner discovery process.
2. Please post a screenshot or listing of your **_/usr/local/lib_** directory.

**Reply by jrodh13** [reply]
From terminal, I see it found the API, but it looks like it was unrecognized. I'm using version 3.15.

2024-06-01 08:39:48.519 INFO  i.g.d.s.t.s.api.SDRplay - API library - open status: Success  [11MB/34MB 33%]
2024-06-01 08:39:48.523 WARN  i.g.d.s.t.s.api.Version - Unrecognized SDRplay API version [3.
[sdrtrunk_app.log]

(https://github.com/user-attachments/files/15521822/sdrtrunk_app.log)
15]  [11MB/34MB 33%]

**Reply by jrodh13** [reply]
I just loaded version 3.14 instead and that one works! I wouldn't have thought to check the API until you mentioned it. Thanks for the help, but I guess 3.15 is not working at the moment.

**Comment by DSheirer** [comment, marked as answer]
@jrodh13 SDRPlay API version 3.15 support was just added to sdrtrunk last week.  If you use the latest nightly build, it should work correctly with API version 3.15.  So, if you want to reinstall that 3.15 version and try the latest sdrtrunk nightly build ... please let me know if there are issues?

**Reply by jrodh13** [reply]
Thank you! That worked!

**Comment by Dawit0101** [comment]
<img width="1440" alt="Screenshot 2025-01-23 at 12 47 32" src="https://github.com/user-attachments/assets/24704b3d-7bca-4c11-8f93-cf77efb45e0b" />
I've got this error. Can't play channel. Error: No tuner Available.
